pkg/common/plugin/exec.go

```diff
@@ -92,14 +100,39 @@ func (e *DefaultExecutor) ExecutePlugin(ctx context.Context, pluginPath string,
 		return nil, e.pluginErr(err, stdout.Bytes(), stderr.Bytes())
 	}
 
+	pluginOutputJSON, pluginOutputMsgs := parsePluginOutput(stdout, stderr)
+
+	for _, msg := range pluginOutputMsgs {
+		msg := strings.ToLower(msg)
+		switch {
+		case strings.HasPrefix(msg, "info"):
+			msg = strings.Replace(msg, "info: ", "", -1)
+			logger.GetLogger(ctx, logOpt).Infof("[Plugin] %s", msg)
+		case strings.HasPrefix(msg, "warn"):
+			msg = strings.Replace(msg, "warn: ", "", -1)
+			logger.GetLogger(ctx, logOpt).Warnf("[Plugin] %s", msg)
+		case strings.HasPrefix(msg, "debug"):
+			msg = strings.Replace(msg, "debug: ", "", -1)
+			logger.GetLogger(ctx, logOpt).Debugf("[Plugin] %s", msg)
+		default:
+			fmt.Fprintf(os.Stderr, "[Plugin] %s,", msg)
+		}
+	}
+
 	// Copy stderr to caller's buffer in case plugin printed to both
 	// stdout and stderr for some reason. Ignore failures as stderr is
 	// only informational.
-	if e.Stderr != nil && stderr.Len() > 0 {
-		_, _ = stderr.WriteTo(e.Stderr)
-	}
+	// if e.Stderr != nil && stderr.Len() > 0 {
+	// 	processStdError(ctx, stderr)
```

Comment:
to confirm, are the commented out blocks for testing and to be removed? :)

Reply:
Good catch, just comments that need removed. :) I'll clean them up.